k8s/Makefile

```diff
@@ -69,12 +76,15 @@ k8s-deployments: k8s-wagtail-deployments k8s-celery-deployments k8s-celery-beat-
 k8s-delete-deployments: k8s-delete-wagtail-deployments k8s-delete-celery-deployments k8s-delete-celery-beat-deployments
 
 k8s-wagtail-deployments:
+	env NEW_RELIC_APP_NAME=dev-portal-web-${TARGET_ENVIRONMENT} \
 	j2 app.deploy.yaml.j2 | ${KC} apply -f -
 
 k8s-celery-deployments:
+	env NEW_RELIC_APP_NAME=dev-portal-celery-${TARGET_ENVIRONMENT} \
 	j2 celery.yaml.j2 | ${KC} apply -f -
 
 k8s-celery-beat-deployments:
+	env NEW_RELIC_APP_NAME=dev-portal-celery-${TARGET_ENVIRONMENT} \
```

Comment:
On Kuma, we set this to a blank value since we don't want to track Celery beat (see https://github.com/mdn/infra/blob/8532d61254e5183bd6e5263cd1425e1e336ae099/apps/mdn/mdn-aws/k8s/Makefile#L603). I suspect you would want to do the same? If you do want to track this, I think you'd want a different app name here rather than using the same one as you used for the `k8s-celery-deployments` target above?

Reply:
Oh good point, fixed this in 0efe30480643d7784c682a4a7733b67cece0c132